UnityProject/Assets/Scripts/Items/Cards/Emag.cs

```diff
@@ -71,6 +71,13 @@ private void SyncCharges(int oldCharges, int newCharges)
 
     }
 
+    public string Examine(Vector3 worldPos)
+    {
+        string desc = "It's a card with a magnetic strip attached to some circuitry. ";
```

Comment:
No need to return the static description as well; interface appends what you return onto the one defined in attributes.
![emag examination](https://user-images.githubusercontent.com/6926225/96825164-fd257700-148c-11eb-88c3-869be48a7725.png)